.infra/index.ts

```diff
@@ -83,10 +85,32 @@ const envVars: Record<string, Input<string>> = {
   redisHost,
 };
 
+const deadLetterTopicName = 'api.v1.dead-letter'
+const deadLetterTopic = new Stream(deadLetterTopicName, { isAdhocEnv, name: deadLetterTopicName, })
+
+const addDeadLetterTopicToWorkers = (workers: Worker[]): Worker[] => {
```

Comment:
I think we should add the dead letter functionality to pulumi-common. It will be useful to reuse in other projects

Reply:
Yes please!